tests/common/wifi/aws_test_wifi.c

```diff
@@ -2446,86 +2543,3 @@ static void prvConnectionTask( void * pvParameters )
 
     vTaskDelete( NULL ); /* Delete this task. */
 }
-
-/**
- * @brief Spawn two threads to connect and disconnect at the same time. This is
- * to verify thread safety in the Wi-Fi driver API.
- */
-TEST( Full_WiFi, AFQP_WiFiSeperateTasksConnectingAndDisconnectingAtOnce )
-{
-    BaseType_t xTaskCreateResult;
-    char cBuffer[ 256 ];
-    int16_t sBufferLength = 256;
-    testwifiTaskParams_t xTaskParams[ testwifiNUM_TASKS ];
-    uint16_t usIndex;
-    uint32_t ulResultEventMask;
-
-    if( TEST_PROTECT() )
-    {
-        /* Create an event group for tasks finishing testing. */
-        xTaskFinishEventGroupHandle = xEventGroupCreate();
-        configASSERT( xTaskFinishEventGroupHandle != NULL );
-
-        /* Even group for task connection and disconnection synchronization. */
-        xTaskConnectDisconnectSyncEventGroupHandle = xEventGroupCreate();
-        configASSERT( xTaskConnectDisconnectSyncEventGroupHandle != NULL );
-
-        /* Create the task that connect at the same time. */
-        for( usIndex = 0; usIndex < testwifiNUM_TASKS; usIndex++ )
-        {
-            xTaskParams[ usIndex ].usTaskId = usIndex;
-            xTaskParams[ usIndex ].xWiFiStatus = eWiFiFailure;
-            xTaskCreateResult = xTaskCreate(
-                prvConnectionTask, "WiFiConnectionTask", testwifiTASK_STACK_SIZE,
```

Comment:
Thank you for removing this function. I have the same comment as the other PR, that prvConnectionTask() function above also needs to be removed since it is used only in this test.

Reply:
ok, will update